test_conformance/images/clCopyImage/test_copy_generic.cpp

```diff
@@ -632,5 +620,16 @@ int test_copy_image_generic( cl_context context, cl_command_queue queue, image_d
         return error;
     }
 
+    /**
+     * Call clFinish() to ensure the unmap call completes.
```

Comment:
Good catch! I don't see this comment style elsewhere in the file. I'd suggest
`// Ensure the unmap call completes.`

Reply:
Cheers. I updated it, to conform to style.